src/amuse/datamodel/parameters.py

```diff
@@ -308,8 +307,8 @@ def __getattr__(self, name):
 
     def __setattr__(self, name, value):
         if not name in self._original._mapping_from_name_to_definition:
-            warnings.warn("tried to set unknown parameter '{0}' for a '{1}' object".format(name, type(self._instance()).__name__), exceptions.AmuseWarning)
-            return
+            raise exceptions.CoreException("Could not set unknown parameter '{0}' for a '{1}' object".format(self.name, type(object).__name__))
```

Comment:
self.name?

Reply:
object->self.instance() ?

Reply:
I clearly did a bad job copy/pasting and keeping the correct bits... fixing it now.

Reply:
ok, above should also be self._original

Reply:
where?

Reply:
type(object).__name__ -. type(self._original).__name__
I am not sure if thats really informative (it will be some parameter set type name) but at least it 
will work

Reply:
ok this is outdated, I changed it back to _instance (as it was in the warning). Should it be _original instead?

Reply:
I think so, I am not sure _instance exists, and it is consistent with if statement...

Reply:
yes, I think you're right.